plugin/helpers_commands.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package plugin
+
+import (
+	"strings"
+
+	"github.com/mattermost/mattermost-server/model"
+	"github.com/pkg/errors"
+)
+
+type CommandCallback func(args *CommandArgs, originalArgs *model.CommandArgs) (*model.CommandResponse, *model.AppError)
```

Comment:
I forgot the server also provided a `Context` parameter -- can we include that here too as a first class parameter?